src/Tile.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+function Tile(props) {
+    return (
+        <section>
+            {props.children}
+        </section>
+    );
+}
```

Comment:
Je had de code nog kunnen opschonen met function Tile({ children }), maar aangezien je alleen children gebruikt snap ik je keuze. Je had ook meer elementen in het component kunnen zeten, zoals title / h2 en de alt tekst.